mypy.ini

```diff
@@ -0,0 +1,37 @@
+[mypy]
+show_column_numbers = true
+warn_unused_ignores = true
+; Target the oldest supported version in editors
+python_version = 3.7
+
+strict = false
+implicit_reexport = true
+
+; Implicit return types !
+; TODO: turn back check_untyped_defs to true. For now this allows us to
+; at least put mypy in place by massively reducing checked code
+check_untyped_defs = false
+disallow_untyped_calls = false
+disallow_untyped_defs = false
+disallow_incomplete_defs = false
+
+; attr-defined: Module has no attribute (modules are dynamic)
+; method-assign: Cannot assign to a method (lots of monkey patching)
+; name-defined: Name "..." is not defined (dynamic modules will be hard to type without stubs, ie: pythoncom.*, leave undefined/unbound to Flake8/Ruff/pyright)
+disable_error_code = attr-defined, method-assign, name-defined
+; TODO: adodbapi should be updated and fixed separatly
+; Pythonwin/Scintilla and Pythonwin/pywin/idle are vendored
+; Ignoring non-public apis for now
+; Duplicate module named "rasutil" and "setup", short-term fix is to ignore
+exclude = .*((build|adodbapi|Pythonwin/Scintilla|Pythonwin/pywin/idle|[Tt]est|[Dd]emos?)/.*|rasutil.py|setup.py)
+
+; C-modules that will need type-stubs
+[mypy-adsi.*,exchange,exchdapi,perfmon,servicemanager,win32api,win32clipboard,win32event,win32evtlog,win32file,win32gui,win32help,win32pdh,win32process,win32ras,win32security,win32service,win32trace,win32ui,win32uiole,win32wnet,wincerapi,winxpgui,_win32sysloader,_winxptheme]
+ignore_missing_imports = True
+
+; verstamp is installed from win32verstamp.py called in setup.py
+; Most of win32com re-exports win32comext
+; Test is a local untyped module in win32comext.axdebug
+; Not sure where the rest of modules/packages come from
+[mypy-verstamp,win32com.*,Test,CallTipWindow,dde,pywin32_system32,]
```

Comment:
Any idea where `CallTipWindow`, `dde` and `pywin32_system32` come from?

Reply:
dde is a module - https://github.com/mhammond/pywin32/blob/main/setup.py#L2025
pywin32_system32 is a hack setup.py does to put the DLLs in a place where they aren't technically a package, eg, https://github.com/mhammond/pywin32/blob/main/setup.py#L337-L339
IIRC, CallTipWindow was a part of idle which we hackily reused -  https://github.com/mhammond/pywin32/blob/4025d7680552beb200688911e3ec87e11acf9365/Pythonwin/pywin/idle/CallTips.py#L38-L41 - I doubt it works at all any more.